src/stan/services/arguments/arg_variational_adapt.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef STAN_SERVICES_ARGUMENTS_ARG_VARIATIONAL_ADAPT_HPP
+#define STAN_SERVICES_ARGUMENTS_ARG_VARIATIONAL_ADAPT_HPP
+
+#include <stan/services/arguments/categorical_argument.hpp>
+#include <stan/services/arguments/arg_variational_adapt_engaged.hpp>
+#include <stan/services/arguments/arg_variational_adapt_iter.hpp>
+
+namespace stan {
+  namespace services {
+
+    class arg_variational_adapt: public categorical_argument {
+    public:
+      arg_variational_adapt() {
```

Comment:
OK, I see this is widespread.  Is this required by the way the arguments are set up now?  If so, it all needs to be changed, but not as part of this pull request.